Mathlib/Mathport/Syntax.lean

```diff
@@ -187,7 +187,6 @@ syntax caseArg := binderIdent,+ (" :" (ppSpace (ident <|> "_"))+)?
 /- M -/ syntax (name := casesType!) "cases_type!" "*"? ppSpace ident* : tactic
 /- N -/ syntax (name := abstract) "abstract" (ppSpace ident)? ppSpace tacticSeq : tactic
 
-/- E -/ syntax (name := eConstructor) "econstructor" : tactic
```

Comment:
You also need to import the new syntax into this file

Reply:
Done, also fixed the trailing white space suggestion.